package.json

```diff
@@ -8,30 +8,67 @@
   },
   "scripts": {
     "build": "hardhat compile",
-    "test": "hardhat test --logs"
+    "test": "hardhat test --logs",
+    "build:watch": "hardhat watch compilation",
+    "test:watch": "hardhat watch test"
   },
   "author": "Prithviraj Murthy",
   "license": "ISC",
   "dependencies": {
-    "@chainlink/contracts": "^0.4.0",
-    "@openzeppelin/cli": "^2.8.2",
-    "@openzeppelin/contracts": "^4.6.0",
-    "@openzeppelin/contracts-upgradeable": "^4.6.0",
+    "@chainlink/contracts": "^0.4.2",
+    "@aave/core-v3": "^1.13.1",
     "@uniswap/lib": "^4.0.1-alpha",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
-    "chai-as-promised": "^7.1.1",
-    "dotenv": "^16.0.1",
-    "hardhat": "^2.9.9",
-    "hardhat-tracer": "^1.1.0-rc.6",
-    "mocha": "^10.0.0",
-    "openzeppelin-solidity": "4.6.0"
+    "@types/yargs": "^15.0.10",
+    "argv": "^0.0.2",
+    "bn.js": "^5.1.1",
+    "dotenv": "^8.2.0",
+    "yargs": "^16.1.1"
   },
   "devDependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.6",
-    "@nomiclabs/hardhat-waffle": "^2.0.3",
-    "@openzeppelin/truffle-upgrades": "^1.15.0",
-    "chai": "^4.3.6",
-    "ethereum-waffle": "^3.4.4",
-    "ethers": "^5.6.8"
+    "@nomiclabs/hardhat-ethers": "^2.0.1",
+    "@nomiclabs/hardhat-etherscan": "^2.1.7",
+    "@nomiclabs/hardhat-ganache": "^2.0.1",
+    "@nomiclabs/hardhat-waffle": "^2.0.0",
+    "@openeth/truffle-typings": "^0.0.5",
+    "@openzeppelin/contracts": "^4.7.2",
+    "@openzeppelin/contracts-upgradeable": "^4.6.0",
+    "@openzeppelin/test-helpers": "^0.5.9",
+    "@typechain/ethers-v5": "^7.0.1",
+    "@typechain/hardhat": "^2.3.0",
+    "@types/chai": "^4.2.21",
+    "@types/debug": "^4.1.5",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.4.13",
+    "@typescript-eslint/eslint-plugin": "^4.29.1",
+    "@typescript-eslint/parser": "^4.29.1",
+    "chai": "^4.2.0",
+    "debug": "^4.2.0",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "^8.3.0 ",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-no-only-tests": "^2.4.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "ethereum-waffle": "^3.2.0",
+    "ethers": "^5.0.22",
+    "hardhat": "^2.4.1",
+    "hardhat-abi-exporter": "^2.3.0",
+    "hardhat-contract-sizer": "^2.0.3",
+    "hardhat-deploy": "^0.7.0-beta.30",
+    "hardhat-gas-reporter": "^1.0.4",
+    "hardhat-tracer": "^1.0.0-alpha.6",
+    "hardhat-watcher": "^2.1.1",
+    "prettier": "^2.3.2",
```

Comment:
prettier can be removed from the package.json do we really  need that in our project